tests/test_api/test_endpoints.py

```diff
@@ -1026,7 +1027,9 @@ def test_find_related_objects_for_workflow_execution__returns_related_workflow_e
     )[0]
 
     # Create a second `WorkflowExecution` that is related to the first one.
-    data_object_b = faker.generate_data_objects(1, has_output=workflow_execution_a["id"])[0]
+    data_object_b = faker.generate_data_objects(
+        1, has_output=workflow_execution_a["id"]
+    )[0]
```

Comment:
Has my version of `black` diverged from yours / the one used by GH CI? Hmm...

Reply:
Hmm, my version was outdated (I was using `24.10.0`, and am now using `25.1.0` after a `make init`), but a subsequent run of `make black` left all files unchanged.